src/Security/Handler/RoleSecurityHandler.php

```diff
@@ -125,4 +116,51 @@ private function isAnyGranted(array $attributes, ?object $subject = null): bool
 
         return false;
     }
+
+    /**
+     * @param array<string|Expression> $attributes
+     */
+    private function hasOnlyAdminRoles(mixed $attributes): bool
```

Comment:
```suggestion
    private function hasOnlyAdminRoles(array $attributes): bool
```
according to the phpdoc